### ISSUE TITLE
Feature Request: Manage fix integrated browse/search facets

### ISSUE THREAD

**Issue by CB-HAL** [IQSS/dataverse#10791]
**Overview of the Feature Request**
It is possible to choose which metadata fields will be used as browse/search facet, but there are some facets where this isn’t possible e.g. Dataverse Category, Publication Year, Language, Publication Status, File Type, File Tag, Access
It would be useful to decide over the use of these facets.
**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
Sysadmin

**Comment by pdurbin** [comment]
@CB-HAL yes, this has been an issue for a long time. Basically, it's possible to hide or show facets that come from the metadata blocks (which are used by datasets) but the others (mostly at the collection or file level) are hard coded.
